src/RCDEF/rcdef.cpp

```diff
@@ -263,7 +263,7 @@ struct SWTABLE	// terminate w/ last array entry of NULL, default/not found indic
 	int val;
 };
 //-----------------------------------------------------------------------------
-int looksw(			// string/word table lookup, case insensitive
+int looksw_rc(			// string/word table lookup, case insensitive
```

Comment:
What is the difference between this function and the other function with the same name?

Reply:
No difference at all. Here are the two locations [rcdef.cpp/looksw](https://github.com/cse-sim/cse/blob/f93628bf1240adf3d032dd9d270aa6de706f4f04/src/RCDEF/rcdef.cpp#L266), [lookup.cpp/looksw](https://github.com/cse-sim/cse/blob/f93628bf1240adf3d032dd9d270aa6de706f4f04/src/lookup.cpp#L80). I believe the intent was to remove `lookup.h` as a dependency of rcdef.

Reply:
But it is a dependency, hence the redefinition error. You are making decisions here without understanding the full context. In this case, you should ask if we really need separate definitions and get a definitive answer. I am not sure if there is any value in keeping them both, so I will defer to @chipbarnaby. I would imagine we can remove the definition in rcdef.cpp.

Reply:
I don't recall the full context either.  I thought I got rid of the lookup.cpp dependency by duplicating this little function.  It worked on MSVC, but I guess not on GCC.  If it is a duplicate definition (without the _rc), then lookup.cpp must still be linked (?).

In general, I don't think it is worth a huge amount of work to achieve absolutely pure non-DRYness, especially in an internal tool like RCDEF.  It is obviously a matter of judgement, but at some point, just do what is needed to make it work, IMHO.

Maybe I'll try to build this on MSVC to understand the issue.

Reply:
I believe it is linked through messages and xiopak.